### PR TITLE
Add maintainers document and code of conduct

### PR DESCRIPTION
**Pull request by breakpointer**
# Description

Drafted maintainers document and added a project code of conduct.

## Motivation and Context

We need to communicate how to be a maintainer of this project and what activities for which maintainers are responsible. Code of conduct was added to make the community aware of our behavior guidelines. It comes from [Contributor Covenant](https://www.contributor-covenant.org/)

## How Has This Been Tested
It has not been tested but will be as the project continues to grow and change. These documents will be updated as needed.

## Types of changes

Read me's

### CODE REVIEW

**Comment by skarred14** [comment]
lgtm

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# How to stop being a maintainer?
+
+If you are a maintainer you can be voted out if any of the following occur: 
+
+ - You stop reviewing PR's, responding to messages, answering emails, and/or generally ghost the project.
+ - You are disrespectful towards anyone in the community and/or involved in the project in some way.
+ - You violate [the code of conduct](./CODE_OF_CONDUCT.md)
+ 
+A simple majority of maintainers in attendance is required to remove a maintainer, but the TSC may be brought in to arbitrate, if the maintainer to be removed or any other party wishes to dispute the action. Two-thirds of all current maintainers constitute a quorum for a meeting involving a question of removal.
```
Comment (bitwiseguy) [inline MAINTAINERS.md]: I would re-word this. 

Two-thirds of all current maintainers constitute a quorum for a meeting involving a question of removal. A simple majority vote from maintainers attending the meeting is required to remove a maintainer, but the TSC may be brought in to arbitrate if the maintainer to be removed or any other maintainer wishes to dispute the action.

Comment (breakpointer) [reply MAINTAINERS.md]: I also think that we should mention that if you want to stop being a maintainer you can leave the role at anytime by notifying the other maintainers.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ 
+
+# Who are the current maintainers?
+
+  - [Sam](https://github.com/bitwiseguy) 
+  - [Patrick](https://github.com/pmacom)
+  - [Kartheek](https://github.com/skarred14)
+  - [Brian](https://github.com/breakpointer)
+  - [John](https://github.com/humbitious)
+
```
Comment (Souptacular) [inline MAINTAINERS.md:22]: Add [Hudson](https://github.com/Souptacular)

File: README.md [hunk 3]
```diff
@@ -83,8 +83,16 @@ optionally `npm run clean` to clean out any `node_modules` folders installed by
 
 # How to contribute?
```
Comment (skarred14) [inline README.md:84]: Not sure if this is the right section, but here are a few additional things, that maintainer(s) could do to reinforce standards and code discipline as part of the responsibilities
- Ensure branch naming conventions: some good references here, https://gist.github.com/digitaljhelms/4287848
- Ensure linting conventions are followed for committing changes
- Ensure PR naming conventions
- Ensure end to end CI process is built in
Feel free to add any other

**Pull request closed without merging**